lib/phoenix_slime.ex

```diff
@@ -1,2 +1,42 @@
 defmodule PhoenixSlime do
+  @doc """
+  Provides the `~l` sigil with HTML safe Slime syntax inside source files.
+
+  Raises on attempts to interpolate with `\#{}`. Use `~L` to interpolate.
```

Comment:
I think it is misleading, `~L` does not allow to interpolate values, instead interpolation processed by slime parser

Reply:
Ah I see your point @Rakoth. Do you think we should remove the `~l` or just rephrase the documentation?

Reply:
It is now matches the behavior of the `~E` sigil, so lets just rephrase the documentation

Reply:
Great!